docs/jwst/photom/reference_files.rst

```diff
@@ -23,6 +33,7 @@ dependent. The current row selection criteria are:
 * MIRI:
    - Imager (includes LRS): Filter and Subarray
    - MRS: Does not use table-based reference file
+   - IFU: Band
```

Comment:
Remove this line. For MIRI, MRS=IFU and as the line above indicates the MRS mode does not use a table-based photom ref file and hence no row selection is done (so this section is irrelevant to MRS).